src/python/WMCore/WMSpec/Persistency.py

```diff
@@ -64,18 +65,21 @@ def load(self, filename):
         if not urlparse(filename)[0]:
             filename = 'file:' + filename
             handle = urlopen(Request(filename, headers={"Accept": "*/*"}))
-            self.data = pickle.load(handle)
+            # FIXME: once both central services and WMAgent are in Py3, we can remove protocol=0
+            self.data = pickle.load(handle, protocol=0)
```

Comment:
I have not checked the report from Jenkins, but `pickle.load()` does not need to specify the protocol version, it infers it from the pickle file/stream itself

Reply:
Oh, you're right! Let me remove it.

Reply:
perfect, thanks! :)